include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -46,9 +46,6 @@ class logging_resource_adaptor final : public device_memory_resource {
    *
    * The logfile will be written using CSV formatting.
    *
-   * If the directories specified in `filename` do not exist, they will be
-   * created.
-   *
    * Truncates `filename` if it already exists.
```

Comment:
Truncates the file's name, or it's contents? Do you mean overwrites the contents?

Reply:
Truncating a file means overwriting the contents: https://en.cppreference.com/w/cpp/io/ios_base/openmode

Reply:
In english, truncate means "to cut short", which is not what you mean here. So I think this documentation is ambiguous and seems like terminology specific to std::ios_base.

```suggestion
   * Overwrites the contents of the file `filename` if it already exists.
```